### PR TITLE
Homepage revamp

### PR DESCRIPTION
**Pull request by martenson**
includes fix for broken responsiveness

please see commits

ping @nekrut 
screenshot 2017-02-17 12 37 05
screenshot 2017-02-17 12 37 13

### CODE REVIEW

File: _includes/main_footer.html [hunk 1]
```diff
@@ -1,3 +1,6 @@
 <footer>
-     <p class="view">View the Project on GitHub: <small><a href="{{ site.github_repository }}">{{ site.github_repository }}</small></a></p>
+  <p>
+    <a href="https://new.galaxyproject.org/teach/gtn/">Galaxy Training Network</a> helps researchers find online training material, and trainers in their geographic area, and helps advertise training events as well.
```
Comment (nsoranzo) [inline _includes/main_footer.html]: Add `The ` before `<a href="https://new.galaxyproject.org/teach/gtn/">Galaxy`

File: _includes/main_header.html [hunk 2]
```diff
@@ -1,37 +1,7 @@
...
-		</tr>
-	</table>
-
-
-
-    <p><a href="https://gitter.im/Galaxy-Training-Network/Lobby?utm_source=badge&amp;utm_medium=badge&amp;utm_campaign=pr-badge&amp;utm_content=badge"><img src="https://camo.githubusercontent.com/66dbecf63f6c9c68127a18afaca4790a7eb6f1c1/68747470733a2f2f6261646765732e6769747465722e696d2f47616c6178792d547261696e696e672d4e6574776f726b2f747261696e696e672d6d6174657269616c2e737667" alt="Gitter" data-canonical-src="https://badges.gitter.im/Galaxy-Training-Network/training-material.svg" style="max-width:100%;"></a> </p>
-
+<header>
+  <h1>{{ site.title }}</h1>
+  <p>Collection of tutorials developed and maintained by worldwide Galaxy community.</p>
```
Comment (nsoranzo) [inline _includes/main_header.html]: `by worldwide` -> `by the worldwide`

File: _layouts/home.html [hunk 3]
```diff
@@ -74,16 +74,17 @@
...
+      <a href="http://www.sfb992.uni-freiburg.de/"><img src="https://raw.githubusercontent.com/bgruening/presentations/bce348bb606c312d531c479e63a66efc2bc38d44/shared/resources/img/MEDEP.jpg" width="15%" style="max-width:100%;"></a>
+      <a href="https://www.ie-freiburg.mpg.de"><img src="https://raw.githubusercontent.com/bgruening/presentations/master/shared/resources/img/14_MPI_IE_logo_mit_180.gif" width="15%" style="max-width:100%;"></a>
+      <a href="https://www.uni-freiburg.de/"><img src="https://raw.githubusercontent.com/bgruening/presentations/a2e38e4b007994af798320db3a0131c4bb891c0e/shared/resources/img/logo_freiburg.jpg" width="15%" style="max-width:100%;"></a>
+      <br>
+      <a href="https://wiki.galaxyproject.org/Teach/GTN"><img src="./shared/images/GTNLogo1000.png" width="15%" style="max-width:100%;"></a>
+      <a href="https://www.france-bioinformatique.fr/"><img src="https://www.france-bioinformatique.fr/sites/default/files/ifb-logo_1.png" width="15%" style="max-width:100%;"></a>
     </p>
-
-    <p>You want to help us on this project? Please, see the <a href="CONTRIBUTING.md"><code>CONTRIBUTING</code></a> file.</p>
```
Comment (nsoranzo) [inline _layouts/home.html]: This line linking to CONTRIBUTING.md has been left out in the reorganisation, worth to save it?

Comment (martenson) [inline _layouts/home.html]: We have link to the github repo which has contributing listed on the page. I dropped it on purpose.

Comment (nsoranzo) [inline _layouts/home.html]: Do you mean the "Fork me on GitHub" image? Works for me.

Comment (bgruening) [inline _layouts/home.html]: I think this line is still important, please note that the audience of this training material might not even now what "Fork me on github" means.

Comment (martenson) [inline _layouts/home.html]: I added it back.

**Comment by bgruening** [comment]
@martenson we should also make some images small, like the GTN logo. It takes ages to load.

**Pull request merged**